scanpy/testing/_pytest/marks.py

```diff
@@ -1,40 +1,54 @@
 from __future__ import annotations
 from importlib.util import find_spec
+from typing import TYPE_CHECKING
 
-import pytest
+if TYPE_CHECKING:
+    import pytest
 
 
 # Mapping from module name to PyPI name
 KNOWN = dict(
-    leidenalg="leidenalg",
-    louvain="louvain",
-    skmisc="scikit-misc",
-    fa2="fa2",
-    igraph="igraph",
-    dask="dask",
-    dask_ml="dask-ml",
-    zarr="zarr",
-    zappy="zappy",
+    leidenalg='leidenalg',
+    louvain='louvain',
+    skmisc='scikit-misc',
+    fa2='fa2',
+    igraph='igraph',
+    dask='dask',
+    dask_ml='dask-ml',
+    zarr='zarr',
+    zappy='zappy',
+    pybiomart='pybiomart',
+    gprofiler='gprofiler-official',
```

Comment:
We generally use `"`

Reply:
https://github.com/scverse/scanpy/pull/2701

Reply:
Great! Let’s merge this one first, then format it so it’s less work.